aw_client/config.py

```diff
@@ -0,0 +1,16 @@
+from configparser import ConfigParser
+
+from aw_core import dirs
+from aw_core.config import load_config
+
+default_client_config = ConfigParser()
+default_client_config["server"] = {
+    "hostname": "localhost",
+    "port": 5600,
+}
+default_client_config["server-testing"] = {
+    "hostname": "localhost",
+    "port": 5666
+}
+
+client_config = load_config("aw-client", default_client_config)
```

Comment:
This whole file is not exposed since it is not in __init__.py so i wouldn't consider this to make much of a difference since it's only exposed to aw_client and not to the watchers using it